README.md

```diff
@@ -7,11 +7,53 @@ and copying out data at when a user requests it (the [`takeout` function]()).
 The functions are  flexible and easy to change to fit the needs of your specific
 users and apps.
 
-### How to play with these functions
+### How to contribute
```

Comment:
Is this really "contribute", as these instructions don't call for contributing back to this repo?  How about something like  "How to use these functions with your app"

Reply:
+1

Reply:
Lines 10 - 17 are instructions for contributing back. The bulk of this PR is  instructions for using the functions in a new app starting below, at line 18.